src/python/WMCore/ReqMgr/Service/Request.py

```diff
@@ -438,6 +442,20 @@ def _handleNoStatusUpdate(self, workload, request_args, dn):
                 msg = "There were unhandled arguments left for no-status update."
                 raise InvalidSpecParameterValue(msg) from ex
 
+        # make call to MSTransferor
+        try:
+            # http POST request to /ms-transferor/data with {'workflow': <workflow>} payload
+            if self.msTransferorUrl:
+                rurl = f"{self.msTransferorUrl}/data"
+                makeHttpRequest(rurl, {'workflow': workload.name()}, method='POST')
+                cherrypy.log('Updated workflow name "{}" in MSTransferor'.format(workload.name()))
+            else:
```

Comment:
Please remove this if/else statement and just make the call.
I think it's safe to assume that MSTransferor url will be defined, otherwise we would be running ReqMgr2 service with a broken and limited feature set.

Reply:
ok, done